src/lj_ccall.c

```diff
@@ -983,7 +1017,7 @@ static int ccall_set_args(lua_State *L, CTState *cts, CType *ct,
     CTypeID did;
     CType *d;
     CTSize sz;
-    MSize n, isfp = 0, isva = 0;
+    MSize n, isfp = 0, isva = 0, onstack = 0;
```

Comment:
I think we'd better guard this variable with the macro as well otherwise we may get C compiler warnings for variables set but not used on other architectures.